### ISSUE TITLE
Enhance a11y for `Breadcrumb`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5401]
name: Breadcrumb

### Description
6 accessibility violation was found in the `Breadcrumb`.

#### [Components / Breadcrumb / Variant](https://yamada-ui.github.io/yamada-ui/?path=/story/components-breadcrumb--variant)

Received:
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (landmark-unique)

Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/landmark-unique?application=axeAPI

#### [Components / Breadcrumb / Size](https://yamada-ui.github.io/yamada-ui/?path=/story/components-breadcrumb--size)

Received:
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (landmark-unique)

Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/landmark-unique?application=axeAPI

#### [Components / Breadcrumb / Separator](https://yamada-ui.github.io/yamada-ui/?path=/story/components-breadcrumb--separator)

Received:
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (landmark-unique)

Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/landmark-unique?application=axeAPI

#### [Components / Breadcrumb / CustomEllipsis](https://yamada-ui.github.io/yamada-ui/?path=/story/components-breadcrumb--custom-ellipsis)

Received:
ARIA hidden element must not be focusable or contain focusable elements (aria-hidden-focus)

Fix all of the following:
  Focusable content should have tabindex="-1" or be removed from the DOM

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-hidden-focus?application=axeAPI

#### [Components / Breadcrumb / CustomEllipsis](https://yamada-ui.github.io/yamada-ui/?path=/story/components-breadcrumb--custom-ellipsis)

Received:
ARIA hidden element must not be focusable or contain focusable elements (aria-hidden-focus)

Fix all of the following:
  Focusable content should have tabindex="-1" or be removed from the DOM

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-hidden-focus?application=axeAPI

#### [Components / Breadcrumb / CustomEllipsis](https://yamada-ui.github.io/yamada-ui/?path=/story/components-breadcrumb--custom-ellipsis)

Received:
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (landmark-unique)

Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/landmark-unique?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
